### ISSUE TITLE
nwtest.config.js reference in README should be nwtest.config.json.

### ISSUE THREAD

**Issue by gbmhunter** [vasanthps/nw-test-runner#3]

**Comment by gbmhunter** [comment]
I believe this is what is causing https://github.com/vasanthps/nw-test-runner/issues/2,

**Comment by mnemanja** [comment]
Agreed. In the source code, the index.js states that it expects a `config = "nwtest.config.json";`

**Comment by vasanthps** [comment]
Sorry. nw-test-runner is deprecated. Another improved version is developed and maintained in https://github.com/Sachinmanoj/nwjs-test-runner
